layouts/partials/post_meta.html

```diff
@@ -1,15 +1,10 @@
-{{- if not .Date.IsZero }}
-<div class="meta__item-datetime meta__item">
-	<svg class="meta__icon icon icon-time" width="16" height="14" viewBox="0 0 16 16"><path d="m8-.0000003c-4.4 0-8 3.6-8 8 0 4.4000003 3.6 8.0000003 8 8.0000003 4.4 0 8-3.6 8-8.0000003 0-4.4-3.6-8-8-8zm0 14.4000003c-3.52 0-6.4-2.88-6.4-6.4000003 0-3.52 2.88-6.4 6.4-6.4 3.52 0 6.4 2.88 6.4 6.4 0 3.5200003-2.88 6.4000003-6.4 6.4000003zm.4-10.4000003h-1.2v4.8l4.16 2.5600003.64-1.04-3.6-2.1600003z"/></svg>
-	<time class="meta__text" datetime="{{ .Date.Format "2006-01-02T15:04:05" }}">{{.Date.Format ( .Site.Params.dateformat | default "January 02, 2006")}}</time>
-{{- end }}
-{{- if ne .Date .Lastmod }}
-	<time class="meta__text" datetime="{{ .Lastmod.Format "2006-01-02T15:04:05" }}"> ({{ T "meta_lastmod" }}: {{.Lastmod.Format ( .Site.Params.dateformat | default "January 02, 2006")}})</time>
-{{- end }}
-</div>
-{{- if .Params.categories }}
-<div class="meta__item-categories meta__item">
-	<svg class="meta__icon icon icon-category" width="16" height="16" viewBox="0 0 16 16"><path d="m7 2l1 2h8v11h-16v-13z"/></svg>
-	<span class="meta__text">{{ range $index, $category := .Params.categories }}{{ if gt $index 0 }}, {{ end }}<a class="meta__link" href="{{ "categories/" | relLangURL }}{{ . | urlize | lower }}" rel="category">{{ . }}</a>{{ end }}</span>
-</div>
-{{- end }}
\ No newline at end of file
+{{- $root := . -}}
+{{- with .Param "post_meta" -}}
+	{{- range $field := . -}}
+		{{- $p := printf "post_meta/%s.html" $field -}}
+		{{- partial $p $root -}}
+	{{- end -}}
+{{- else -}}
+	{{- partial "post_meta/date.html" $root -}}
```

Comment:
Can we handle the situation when the user wants to turn off all `post_meta` completely?

Reply:
Hugo treats an empty list as false? If not, the solution is already there. If yes, we could make the default no meta and ship the example config and readme with `post_meta=["date","categories"]`

Reply:
> If yes, we could make the default no meta and ship the example config and readme with post_meta=["date","categories"]

Plus, we need to add the extended description about Post meta in README.md configuration section (something like [Sidebar](https://github.com/Vimux/Mainroad#sidebar)) and remark just below config.toml example (like "Always check that your site's config file matches the latest theme config file format before update").

Reply:
I have just pushed a less breaking alternative. No configuration gives sane defaults (date + category), setting `["none"]` disables.

Reply:
> No configuration gives sane defaults (date + category), setting ["none"] disables.

OK. This is a trick, it can confuse users, but we keep backward compatibility. I think we will fix it later when we accumulate a few more breaking changes.